### ISSUE TITLE
Bots barely ever try to burn biomass

### ISSUE THREAD

**Issue by WhenTanksFly** [ReactiveDrop/reactivedrop_public_src#621]
The Reduction video made me remember the bots _are_ supposed to bring out their flamer and burn biomass close to them. Problem is that only works correctly like once in a few months, and usually they'd ignore biomass even when ordered to walk right next to it with flamer already being their active weapon.

I don't really see what else can I add, other than perhaps go play some missions with biomass in singleplayer and save the demos. Maybe later.

**Comment by BenLubar** [comment]
Yeah it seems like they have a faulty line of sight calculation or something.

**Comment by WhenTanksFly** [comment]
Talking of line of sight...

https://github.com/ReactiveDrop/reactivedrop_public_src/assets/112336094/13301436-e2ec-47f8-b911-38347dd39c98
